docs/nitpick-exceptions

```diff
@@ -15,28 +15,18 @@ py:obj SpectralQuantity
 py:obj SkyCoord
 
 # Super class for Spectrum1D for which we don't control the API links.
+py:class ndcube.mixins.ndslicing.NDCubeSlicingMixin
```

Comment:
I think most of this came from ndcube, so I recreated the list from scratch. Makes it shorter too.